boards/arm/nucleo_f429zi/doc/index.rst

```diff
@@ -197,3 +213,9 @@ This interface is supported by the openocd version included in Zephyr SDK.
 
 .. _STM32F429 datasheet:
    http://www.st.com/resource/en/datasheet/DM00071990.pdf
+
+.. _zephyr partition table:
```

Comment:
This link target doesn't exist.  I suggested a change above to use a ``:ref:`` to link to the correct document location, so you can remove these two lines (217-218)